File: Latihan A/Ram.cpp
class Ram{

	private:
		//atribut private
		int capacity; //kapasitas ram
		int price; //harga ram

	public:

		//konstruktor
		Ram(){

		}

		Ram(int c, int p){
			this->capacity = c;
			this->price = p;
		}

		//getter dan setter atribut capacity
		void setCapacity(int c){
			this->capacity = c;
		}

		int getCapacity(){
			return this->capacity;
		}

		//getter dan setter atribut price
		void setPrice(int c){
			this->price = c;
		}

		int getPrice(){
			return this->price;
		}

		//destruktor
		~Ram(){

		}

};

File: Latihan B/Python/Ram.py
class Ram():

	#atribut private
	iCapacity = 0 #kapasitas ram
	iPrice = 0 #harga ram

	#konstruktor
	def __init__(self, iCapacity = 0, iPrice = 0):
		self.iCapacity = iCapacity
		self.iPrice = iPrice

	#getter dan setter atribut iCapacity
	def setCapacity(self, iCapacity):
		self.iCapacity = iCapacity

	def getCapacity(self):
		return self.iCapacity

	#getter dan setter atribut iPrice
	def setPrice(self, iPrice):
		self.iPrice = iPrice

	def getPrice(self):
		return self.iPrice